index.ts

```diff
@@ -222,4 +226,10 @@ export class SocketIOMetrics<T extends string> {
             return 0;
         }
     }
+
+    private bindNamespaces() {
+        for (const nsp of Object.keys((this.ioServer as any).nsps)) {
```

Comment:
nit: this will throw an exception if `this.ioServer` does not define `nsps`. 
```suggestion
        for (const nsp of Object.keys((this.ioServer as any).nsps ?? {})) {
```

Reply:
Thanks for the suggestion, brought in with latest commit!